### ISSUE TITLE
make sure future commits are more granular and named properly

### ISSUE THREAD

**Issue by greglinch** [Solomonabo/wp-content#17]
Related to this comment:

https://github.com/Solomonabo/wp-content/issues/16

Commits should be for the smallest group of similar changes to make something work. Grouping things together makes it hard to follow you progress.

When you're copy-pasting code from somewhere else, such as open-source code or the parent theme, be sure to add those as separate commits from your customizations and credit them in the commit message as such.

**Comment by Solomonabo** [comment]
Well noted.
